internal/helm/helmadapter/helm2_env_service.go

```diff
@@ -140,10 +142,10 @@ func (h helmEnvService) DeleteRepository(_ context.Context, helmEnv helm.HelmEnv
 	return nil
 }
 
-func (h helmEnvService) PatchRepository(_ context.Context, helmEnv helm.HelmEnv, repository helm.Repository) error {
+func (h helmEnvService) PatchRepository(ctx context.Context, helmEnv helm.HelmEnv, repository helm.Repository) error {
```

Comment:
I forgot to remove this although it's already unused. feel free to remove it in this pr or leave it for later..

Reply:
thx i'll remove it in alater PR